src/LaunchDarkly.ServerSdk.Redis/LaunchDarkly.ServerSdk.Redis.csproj

```diff
@@ -21,7 +21,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="LaunchDarkly.ServerSdk" Version="7.0.0" />
```

Comment:
So, this time I make it work with greater than or equal to 8, but less than 8. The previous form worked because dotnet generates binding redirects and would just be happy most of the time. The new one is technically more accurate.

Reply:
I don't really understand this to a great degree. My naive intuition would be that it'd only generate binding redirects within a major version. Is that totally false?

Reply:
It can work even for major versions. But I am not completely sure which conditions each toolchain may consider.

The dependency resolution is really annoying. It defaults to "lowest" versus highest compatible.

So before we had, in some packages `[1.1.0, 3.0.0)` (or something close). It would install 2.0.0 and use it on most platforms. Asp would be OMG IT ISNT 1.1.0, and explode, so you have to manually write the redirect.

Having the range at least makes it clear what we expect.